lib/creditsafe/client.rb

```diff
@@ -16,6 +16,9 @@ module Creditsafe
   class Client
     ENVIRONMENTS = %i(live test).freeze
 
+    RETRY_SLEEP_TIME = 5
+    RETRY_LIMIT = 3
```

Comment:
Is it worth specifying that these retry settings are specifically for authentication errors?